atlassian_jwt_auth/contrib/tests/django/test_django.py

```diff
@@ -0,0 +1,142 @@
+import os
+
+import django
+from django.test.testcases import SimpleTestCase
+from django.test.utils import override_settings
+
+from atlassian_jwt_auth import create_signer
+from atlassian_jwt_auth.contrib.tests.utils import get_static_retriever_class
+from atlassian_jwt_auth.tests import utils
+from atlassian_jwt_auth.tests.utils import RS256KeyTestMixin
+
+
+def create_token(issuer, audience, key_id, private_key):
+    signer = create_signer(issuer, key_id, private_key)
+
```

Comment:
Is this a deviation from the pep8 recommendation? https://www.python.org/dev/peps/pep-0008/#blank-lines